packages/side-nav/src/SideNavItem.tsx

```diff
@@ -21,7 +21,7 @@ const defaultStyle = css`
   position: relative;
   width: 100%;
   margin: unset;
-  padding: 8px ${SIDE_OFFSET}px 8px ${SIDE_OFFSET}px;
+  padding: 6px ${SIDE_OFFSET}px 6px ${SIDE_OFFSET}px;
```

Comment:
Hmm... can you point me to the mocks these values are from? I worked with Nelle on this, and to align with the typography / spacing scale we’re slowly moving to.

Reply:
![designWithSpecs](https://user-images.githubusercontent.com/1130737/74575297-6e9f4400-4f54-11ea-8608-aa3d50f1affb.png)

This is the screenshot from Invision that passed through to me via Harry (and I think Nelle?)

Reply:
As noted in my previous comment, we're discussing how best to handle the length issue, and I'll get back to you about it.